File: RemoteTest.Core/Dto/MeterReadingViewDto.cs
using CsvHelper.Configuration.Attributes;

namespace RemoteTest.Core.Dto
{
    public class MeterReadingViewDto
    {
        public int Id { get; set; }
        public int AccountId { get; set; }
        public DateTime MeterReadingDateTime { get; set; }
        [Format("dd/MM/yyyy HH:mm")]
        public string MeterReadValue { get; set; }

        public string FirstName { get; set; }
        public string LastName { get; set; }
    }
}

File: remotetest.client/src/models/MeterReadingViewDto.ts
export interface MeterReadingViewDto {
  id: number;
  accountId: number;
  meterReadingDateTime: string; // Use string for ISO date/time, or Date if you will parse it
  meterReadValue: string;       // Format can be handled in UI if needed
  firstName: string;
  lastName: string;
}